.snyk

```diff
@@ -0,0 +1,7 @@
+# https://docs.snyk.io/scan-applications/snyk-code/using-snyk-code-from-the-cli/excluding-directories-and-files-from-the-snyk-code-cli-test
+exclude:
+  global:
```

Comment:
I don't think we need this.

I am fine if the lane fails so we know there is a problem and we can contribute to this two projects to fix this issue :)

I can take the controller-runtime one no problem

Reply:
I just check the code for controller runtime and there is not need to fix that one we just need to ignore it.

for the ginkgo one I think we can try to ask for a PR that should be easy I think

in general please use
https://docs.snyk.io/snyk-cli/commands/ignore#description
for the controller-runtime put non expiration as the code is expected
for ginkgo you can add 2 month or something like that and we can try to open a PR for them

Reply:
done

Reply:
I agree with the general strategy, but I'm pretty convinced ginkgo _wants_ to emit a stacktrace in the logs in the reported case.

Reply:
@ffromani I can understand that. So do you suggest I'll reword the reason in ginkgo to "the code is expected." also?

Reply:
I don't think it's a blocking issue and we can revisit in 2 months